packages/strapi-plugin-settings-manager/admin/src/components/EditFormSection/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+*
+* EditFormSection
+*
+*/
+
+import React from 'react';
+import { map, isEmpty } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+// HOC Form
+import WithFormSection from 'components/WithFormSection';
+// nested form
+import EditFormSectionNested from 'components/EditFormSectionNested';
+
+class EditFormSection extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    const sectionName = isEmpty(this.props.section.name) ? '' : <FormattedMessage {...{id: this.props.section.name}} />;
+    // get the styles from the WithFormSection HOC
+    const styles = this.props.styles;
```

Comment:
You don't need to create this variable. It will be okay if you have to format it or the values. Maybe it's an anticipation for a next iteration (not sure about my comment)?